sga/templatetags/math.py

```diff
@@ -1,14 +1,21 @@
+"""
+Custom math templatetags
+"""
+
 from django import template
 
 register = template.Library()
 
+
 @register.filter
 def subtract(value, arg):
-    """ Subtracts the arg from the value """
+    """
+    Subtracts the arg from the value
+    """
     try:
         return int(value) - int(arg)
     except (ValueError, TypeError):
         try:
             return value - arg
-        except Exception:
-            return ""
\ No newline at end of file
+        except Exception:  # pylint: disable=broad-except
+            return ""
```

Comment:
This implementation was adapted directly from the [add() function in Django's defaultfilters.py](https://github.com/django/django/blob/master/django/template/defaultfilters.py#L718).

```
@register.filter(is_safe=False)
def add(value, arg):
    """Adds the arg to the value."""
    try:
        return int(value) + int(arg)
    except (ValueError, TypeError):
        try:
            return value + arg
        except Exception:
            return ''
```